### PR TITLE
Add popup menu: Release

### PR DESCRIPTION
**Pull request by manq2010**
Portfolio: details popup window
--- 
In this release, the following was done:
* Added popup window for both mobile and desktop

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -424,6 +424,66 @@ <h2 class="footer-heading">Get started</h2>
         </fieldset>
```
Comment (HAMMAS-SALEEM) [inline index.html:424]: - [ ] Kindly comment all your project cards from your html file generate these cards from your **JavaScript** dynamically by using the array of objects which you have made in your **JavaScript** file :+1:

File: index.html [hunk 2]
```diff
@@ -424,6 +424,66 @@ <h2 class="footer-heading">Get started</h2>
...
+              <span>Next project</span>
+              <a href="#">
+                <img src="img/ic_arrow_right-modal.svg" alt="right modal">
+              </a>
+            </div>
+
+          </div>
+        </div>
+      </div>
+    </section>
```
Comment (HAMMAS-SALEEM) [inline index.html:486]: - [ ] Kindly remove modal card from your html file generate this modal card from your **JavaScript** dynamically by using the array of objects which you have made in your **JavaScript** file :+1:

File: index.html (hunk 1 above)
Comment (HAMMAS-SALEEM) [inline index.html:424]: ### Your Desktop Modal
image

### Your Mobile Modal
image


- [ ] You have created your modal window but your modal window design is not very fit on the screen cuz you can see there is a horizontal scrollbar. Kindly fix your design in both desktop and mobile version :+1:

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

## Hey @manq2010, @mkaya13, @yonas44 :wave:

## Highlights :1st_place_medal:

:heavy_check_mark: Descriptive PR Title
:heavy_check_mark: Descriptive PR Summary
:heavy_check_mark: Following Linters
:heavy_check_mark: Following some requirements

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (manq2010) [reply index.html:424]: Please see how the design looks on my side... I used `inspect` and set it to the correct one with i.e for desktop set it the given `1440px` to mimic the figma design  :blush: :blush:

[My previous reviewer mentioned I do the same](https://github.com/manq2010/my-portfolio/pull/6) :blush: :blush:

image

Desktop Modal opened
---
image

Mobile Version
---
image

image

File: index.html (hunk 2 above)
Comment (manq2010) [reply index.html:486]: I think this code might improve the code to make it dynamic which is achieved by not putting the modal part of HTML on **javascript**. When I put the modal part of HTML on **javascript**, the callback events are not responsive. Perhaps you can assist with a different approach if the one just submitted does not meet the requirements.

```js
buttons.forEach((btn, index) => {
  btn.addEventListener('click', () => {
    modalContainer.style.display = 'block';
    title.innerHTML = cardContainer[index].title;
    modalPicture.style.background = `url(
    ${cardContainer[index].image[0]}) no-repeat center center/ cover`;
    modalPictureDesktop.style.background = `url(
      ${cardContainer[index].image[1]}) no-repeat center center/ cover`;

    projectDetails.innerHTML = cardContainer[index].projectDetails;

    while (codingLanguages.hasChildNodes()) {
      codingLanguages.removeChild(codingLanguages.firstChild);
    }

    cardContainer[index].codingLanguages.map((lang) => {
      const language = document.createElement('li');
      language.innerHTML = lang;
      codingLanguages.appendChild(language);
      return '';
    });

    link1.setAttribute('href', cardContainer[index].liveDemo);
    link2.setAttribute('href', cardContainer[index].liveSource);
  });
});

```

**Comment by manq2010** [comment]
For Reference
---

**Desktop view**
image

**Comment by manq2010** [comment]
The figma one is also at 1440px width
---

image

**Review by ahmetbozaci — COMMENTED** [review]
### Hello @manq2010 
### _This is an additional review as a result of the second opinion request._ 


**I checked your project at 1440px width.**

- This is what looks like after this commit `Update JS to be dynamic`. It is not centered. Near to right side.
### Desktop version
image

- Updated version(after this commit `Update card position`). It is not centered. Near to the left side.
### Desktop version
image
**As you can see the desktop version is not still centered. No issue with the mobile version.**

Kindly work on this issue and submit it for review after you have done.







___

### Cheers and Happy coding! 💻 👏
____
[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Pull request merged**